nectar/config.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
+from builtins import object
 
 import os
 import tempfile
 
 import requests.auth
 
+temp_files = []
```

Comment:
@frenzymadness  well I just find out that it is sort of desired behaviour for test https://github.com/frenzymadness/nectar/blob/95196b9e03e8575cb9419b2f9e86f62ab789c407/test/unit/test_config.py#L63-L70. (It's still an aesthetic bug, but that's not concern of this PR) Just put it back how it is and then we will be ready to merge this.